benchmark/attention_inference/configs/__init__.py

```
@@ -0,0 +1,17 @@
+import pkgutil
+
+
+def list_configs():
+    return sorted(m.name for m in pkgutil.iter_modules(__path__) if not m.name.startswith("_"))
+
+
+def load_config(name: str):
+    try:
+        module = importlib.import_module(f".{name}", __package__)
+    except ModuleNotFoundError:
+        raise ValueError(f"Unknown config '{name}'. Available: {', '.join(list_configs())}")
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)